index.js

```diff
@@ -1,66 +1,87 @@
-let arrayOfBooks = [];
+const documentVariables = {
+  booksCatalog: [],
+  listOfBooks: document.getElementById('book-list'),
+  inputTitle: document.getElementById('title-name'),
+  inputAuthor: document.getElementById('author-name'),
+  addBookButton: document.getElementById('add'),
+};
 
-const addButton = document.getElementById('add');
-const bookList = document.getElementById('book-list');
-const titleInput = document.getElementById('title-name');
-const authorInput = document.getElementById('author-name');
-
-function addBook() {
-  const newBook = {
-    title: titleInput.value,
-    author: authorInput.value,
-  };
-
-  if (newBook.title !== '' && newBook.author !== '') {
-    arrayOfBooks.push(newBook);
+class BookCatalog {
+  constructor() {
+    documentVariables.addBookButton.addEventListener('click', this.addBook);
+    window.addEventListener('load', this.loadBooks);
+    window.addEventListener('unload', this.saveBooks);
   }
-}
-
-function removeBook(event) {
-  const bookKey = event.currentTarget.parentElement;
-  const bookContainer = bookKey.parentElement;
 
-  const filteredArray = arrayOfBooks.filter(book => book.title !== bookKey.children[0].innerText
-    || book.author !== bookKey.children[1].innerText);
-
-  arrayOfBooks = filteredArray;
-
-  bookContainer.removeChild(bookKey);
-}
+  // eslint-disable-next-line class-methods-use-this
+  addBook() {
+    if (documentVariables.inputTitle.value !== '' && documentVariables.inputAuthor.value !== '') {
+      const newBook = {
+        title: documentVariables.inputTitle.value,
+        author: documentVariables.inputAuthor.value,
+      };
+      documentVariables.booksCatalog.push(newBook);
+    }
+  }
 
-function loadBooks() {
-  const strBooks = window.localStorage.getItem('books');
+  // eslint-disable-next-line class-methods-use-this
+  loadBooks() {
+    const strBooks = window.localStorage.getItem('books');
+    if (strBooks !== null) {
+      documentVariables.booksCatalog = JSON.parse(strBooks);
+      /* eslint-disable */
+  
+      for (const book of documentVariables.booksCatalog) {
+        const bookCont = document.createElement('div');
+        const titleAuth = document.createElement('div');
+        const titleAuthClasses = 'd-flex justify-content-around'.split(' ');
+        titleAuth.classList.add(...titleAuthClasses);
+        const bookName = document.createElement('span');
+        bookName.classList.add('d-inline');
+        const authorName = document.createElement('span');
+        authorName.classList.add('d-inline');
```

Comment:
- [ ] Please add headers to the span so the title has a title and the author have one also.

Reply:
Well noted